payload-schemas/api.github.com/common/committer.schema.json

```diff
@@ -1,18 +1,34 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Did you mean to change all the formatting and the sorting of the keys?

Reply:
> Did you mean to change all the formatting and the sorting of the keys?

It was an auto-format from my IDE, reverting this 🙏🏽